poc_mobile_verifier/src/cli/generate.rs

```diff
@@ -18,37 +16,61 @@ pub struct Cmd {
     #[clap(long)]
     before: NaiveDateTime,
     #[clap(long)]
-    input_bucket: String,
+    periods: i32,
```

Comment:
what are `periods`? can we document this with a comment if there's no descriptive type that better conveys what this is? is it a number of hours? an index of time within a duration?

Reply:
a better name would be `number_of_verification_periods`, so it is the number of verification epochs for the given duration

Reply:
has been removed, so resolving